install.sh

```diff
@@ -12,89 +12,135 @@ binary_file="${install_to}/wpscan-analyze"
 set -e
 
 # Cd to temp and try to create/delete file
-cd /tmp/
+cd /tmp/ || ( mkdir -p tmp && cd tmp )
```

Comment:
That's odd. Why should `cd /tmp` fail?

Reply:
Yes sometimes you don't have access to `/tmp/`, it depends of your environment

Reply:
Out of curiosity, which system don't have /tmp or prohibit access to it?

Reply:
Terminal emulation on android for exemple

Reply:
LOL. Okay. Not sure we should target these systems though while complicating our main targets. What do you think?